wen-restart/src/wen_restart.rs

```diff
@@ -501,11 +502,7 @@ pub(crate) fn generate_snapshot(
         let parents = new_root_bank.parents();
         banks.extend(parents.iter());
 
-        let _ = my_bank_forks.send_eah_request_if_needed(
-            my_heaviest_fork_slot,
-            &banks,
-            accounts_background_request_sender,
-        )?;
+        let _ = snapshot_controller.send_eah_request_if_needed(my_heaviest_fork_slot, &banks)?;
```

Comment:
Maybe a small glimpse into what a one-off snapshot request could look like down the road.